### ISSUE TITLE
Update Facility Owner Values (NN)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#1910]
### As a ROW permit analyst, I'd like the Facility Owner Info Updated once the COA Division info is merged into it. Some of the infos need to be removed, have their name updated and the abbreviation changed.
**Must be worked with #2397**

**StakeHolders:** Jaclyn L, Isaiah L, Paloma
**DTS Manager:** Nate

**Acceptance Criteria:**
- Merge Franchise Owners/Project Sponsors Info and COA Division Info into one (See List attached) Infos: 76765 & 75943
- Keep Franchise Owners/Project Sponsors Info. Delete COA Division info, Delete the Duplicates
- They need to be added to the EX for all SubTypes
- They need to be added to the  RW folder for SubType TURP
- Need to add CPO Corridor Program Office and BlueBonnet Electric to 76765 & 75943

**Notes:**
- Info field pick values need to be updated
- EX & RW folder: Infos are 76765 & 75943. 
- They just need one field, keep 76765 (Facility Owner/Project Sponsor), and update the pick values with list attached (More revisions to come). 
- Folder type(s): EX SubType: All AND RW SubType: TURP

**List:**
[Facility Owner Drop Down Menu.xlsx](https://github.com/cityofaustin/atd-data-tech/files/5092531/Facility.Owner.Drop.Down.Menu.xlsx)

Note: Data clean up happening currently.

**Ready for Development?** No

*Migrated from [atd-amanda #204](https://github.com/cityofaustin/atd-amanda/issues/204)*

**Comment by SurbhiBakshi** [comment]
1. Update facility owner values 
2. Facility Owners list from Info Tab 
3. Greg or Isiah are good contacts to get updated list.

**Comment by SurbhiBakshi** [comment]
Wrote to Jaclyn L, Isaiah L, Greg P to check if they have an updated list of facility owners.


![Facility_owner_Amanda.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/df077fcd-7787-486b-b2d9-3ce11e0979c1)

**Comment by SurbhiBakshi** [comment]
[https://app.zenhub.com/files/140626918/5c9818bd-7316-4456-8add-da1ae4d22738/download](https://app.zenhub.com/files/140626918/5c9818bd-7316-4456-8add-da1ae4d22738/download)

**Comment by SurbhiBakshi** [comment]
[https://app.zenhub.com/files/140626918/95a6ced1-b3d9-41b0-a186-b612d007a27a/download](Facility Owner List - needs to be reviewed)

**Comment by SurbhiBakshi** [comment]
As per Greg P, Hey there,
- Paloma and team need to see this list.
- need to make sure the pull down menus are the same as the EX and RW sub: Temp Use of ROW. The look up tables need to the same table.
(For some reason, I remember CTM creating an additional look up tables instead of using the same table for all ROW permits. That may have been fixed, but if not, we need to make sure EX – all subs, RW-Temp use of ROW (TURP) and UC all use the same table . Please disregard if this has already been resolved. It’s been awhile since I have been involved with the ROW permitting AMANDA stuff. 😊)

**Comment by SurbhiBakshi** [comment]
Email from Paloma - Good evening all,

 

I ran the list past a few experts and they offered the following …

 

Possible adds:

Hays County Texas

Blanco County Texas

ATD – AMD

CIP

 

Possible deletes:

Abovenet - I've never seen them used and they were acquired by Zayo in 2012.

Enterprise Texas Pipeline, Kinder Morgan Pipeline, El Paso Field Svcs - never see them used

Convention Center, so that there’s only one ACC

 

Other comments: 

Kinder Morgan is the pipeline company trying get approval and acquire land to run the Gas line though the Hill Country, Buda, and Kyle at this time though.

I would like to see some of the groups combined, for example only one ATT.

 

Thank you all for the input!

**Comment by SurbhiBakshi** [comment]
Email from Gregory P - 

Good stuff Paloma. 
Burning the candle.late I see.

Here are my comments:

Kinder Morgan, Enterprise, and EPFS pipelines should stay. They still operate pipelines in COA. We may have some permits for relocations for a couple of these on 2 of the corridors.

ATT cannot be combined. The ATT Texas (dba SWBT) and other ATT's are each recognized as separate entities by the State Public Utility Commission, the City, and FCC. They have to be separated to match each individual company. Each company has to be listed for any legal and/or liability issues.

Isaiah,
I like the idea of the GP matching, but we can't make any revisions to the GP in this project, unless we get sign off from DSD (OB and Roderick) If you think that is something they are willing to do, then will you take the lead and reach out to them? Please note that the names and pull downs should be matching the acronyms from the GP program for each company in our folder IF they have an annual GP. If not, we should coordinate with OB to see what they may use in the future.

Also, for the additions that Paloma suggested, we need to see if they have, or ever had, a GP to see what acronyms we should use.

Thanks all. Sorry for any errors.

**Comment by TracyLinder** [comment]
I need to read through these notes, and compile a final input list.

**Comment by TracyLinder** [comment]
@Nadin-Nader please work w/ SME's to get final input list during sprint 26.

**Comment by Nadin-Nader** [comment]
@Nadin-Nader Get with Greg and Jaclyn

**Comment by Nadin-Nader** [comment]
Sent email to Jaclyn and Greg asking them to revise the spreadsheet and explain it a bit more so that we can make the necessary updates. Email is below and so is the attached zip i sent:

> Hi Jaclyn/Greg,
> 
> Hope you are both doing well. I am reaching out regarding Issue #1910 where you would like us to update the Facility Owner values. I have looked at the spreadsheet attached and found that I need it explained to me as it is a bit confusing.
> We can schedule some time to go over the spreadsheet or if you can go through it and send me an updated copy of what needs to be removed, modified or changed.
> Please let me know as soon as you can so that we can start looking into this and assigning a resource to it.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services 
> Austin Transportation Department

[https://app.zenhub.com/files/140626918/194fe7b0-c18e-4c65-94d0-89eb5fe70835/download](https://app.zenhub.com/files/140626918/194fe7b0-c18e-4c65-94d0-89eb5fe70835/download)

**Comment by TracyLinder** [comment]
Paloma is good w/ the input provided by Jaclyn & Pepper, doesn't need to review it.

**Comment by TracyLinder** [comment]
**Status:** Contact Greg P & Jaclyn to get final input list.

**Comment by TracyLinder** [comment]
@Nadin-Nader I added you as DTS Manager to this one. It's being scoped by Kaarthy during sprint 29.

**Comment by licarir** [comment]
[https://app.zenhub.com/files/140626918/c3c16e61-eddf-461e-a147-82767c9f97ae/download](https://app.zenhub.com/files/140626918/c3c16e61-eddf-461e-a147-82767c9f97ae/download)

**Comment by licarir** [comment]
@TracyLinder @Nadin-Nader

We are going to send the attached ValidInfoValue-Export.txt file above to Paloma and others to get clarification on the changes to be made. If you have any updates to this list, let us know which items need to be deleted/added/updated as well.

@kaarthym5 will set up a meeting with Jaclyn and Paul to get their inputs on the list.

Thank you,
--
Robert

**Comment by Nadin-Nader** [comment]
Sent email to Greg, Jaclyn, Paloma, Kim and Isaiah:

Hello Everyone,


Hope you are all doing well. I am reaching out to you in regards to the request to Update Facility Owner Values. Below I have added the list for you to go over (I have also added the link). Please go over this and let us know if it is correct or if it needs more updating. If a meeting is needed, please let me know as well and I will set one up with the developers.

Once we get the response, we can go ahead and promote the changes.

https://app.zenhub.com/files/140626918/c3c16e61-eddf-461e-a147-82767c9f97ae/download

INFOCODE	INFOVALUE	STAMPDATE	STAMPUSER	ACCESSCODE	INFOVALUE2	INFODESC	INFODESC2	COMMENTS
76765	ACC-C (COA Austin Convention Center - Convention)							
76765	ACC-L (COA Austin Convention Center - Long)							
76765	ACC-P (COA Austin Convention Center - Palmer)							
76765	ACI      (Abovenet Communications)							
76765	AEC     (Atmos Energy)							
76765	AEU     (COA-Austin Energy)							
76765	AFD     (Austin Fire Department)							
76765	ALP      (Alpheus Communications)							
76765	APG     (Apogee)							
76765	ATD      (Austin Transportation Department) 							
76765	ATT       (ATT Texas Communications)							
76765	ATT-L    (ATT Legacy-National side)							
76765	ATT-MÂ  (ATT Metro) 							
76765	AVI       (COA-Aviation Department)							
76765	AWU    (COA-Austin Water Utility)							
76765	CMT     (Capital Metro Transportation)							
76765	CRC     (Circle C HOA-per settlement agreement)							
76765	D17      (Travis County Water District 17)							
76765	EFS     (El Paso Field Services)							
76765	ETP     (Enterprise Texas Pipeline)         							
76765	FLC      (Fiberlight)							
76765	FPL      (FPL FiberNet) 							
76765	GFT      (Google Fiber Texas, LLC)							
76765	GRA     (Grande Communications)							
76765	GTN      (GAATN Communications)							
76765	KMP     (Kinder Morgan Pipeline)							
76765	LV3      (Level 3 Communications)							
76765	MCI      (MCI_Metro Access Communications/Verizon)							
76765	NPZ      (COA-Neighborhood Planning and Zoning Department)							
76765	OTH     (Other)							
76765	PAR     (COA-Parks and Recreation Department)							
76765	PEC     (Pedernales Electric Company)							
76765	PVT      (Private Developer)							
76765	PWD    (COA-Public Works Department)							
76765	QST      (QWEST/CenturyLink)							
76765	SBP      (Special Bikeway Permit-IDIQ)							
76765	STX       (State of Texas)							
76765	SWP     (Special Sidewalk Permit-IDIQ)							
76765	TCTX     (Travis County Texas)							
76765	TGS      (Texas Gas Service)							
76765	TWC     (Time Warner Cable)							
76765	TWT     (Time Warner Telecom)							
76765	UTA      (University of Texas Austin)							
76765	WCTX  (Williamson County Texas)							
76765	WPD    (COA-Watershed Protection Department)							
76765	XOC     (XO Communications)							
76765	ZYO     (Zayo)	


Respectfully,

Nadin A. Nader 
IT Business Systems Analyst
Data and Technology Services 
Austin Transportation Department

**Comment by Nadin-Nader** [comment]
Sent email to everyone and set a meeting for the 18th at 1230pm to go over and refine the requirements and acceptance criteria.

**Status:**
Meeting set for the 18th @1230pm

**Comment by Nadin-Nader** [comment]
@kaarthym5 @TracyLinder List has been added and so has the acceptance criteria. Moving to Backlog.

**LIST:**
[Facility Owner Drop Down Menu.xlsx](https://github.com/cityofaustin/atd-data-tech/files/5092545/Facility.Owner.Drop.Down.Menu.xlsx)

**Status**
Ready to start work

**Comment by TracyLinder** [comment]
We're merging this issue w/ #2397. Closing.
